pages/CreateStory.js

```diff
@@ -8,5 +21,18 @@ export default function NewStoryPage({ addStory }) {
     router.push("/");
   }
 
-  return <NewStoryForm onSubmit={handleStorySubmit} />;
+  return (
+    <>
+      <Header>Create a new story</Header>
+      <Label htmlFor="Click to use Storyteller"></Label>
```

Comment:
Empty label?

Reply:
also you should not use whitespaces in ids, better use `use-storyteller` or similar as an id

Reply:
Changes made.